stdlib/database/sql/src/main/java/org/ballerinalang/sql/Constants.java

```diff
@@ -37,10 +37,9 @@
     public static final String CONNECTOR_ID_KEY = UUID.randomUUID().toString();
 
     public static final String DATABASE_ERROR_DATA = "DatabaseErrorData";
-    public static final String DATABASE_ERROR_CODE = "{ballerina/sql}DatabaseError";
+    public static final String DATABASE_ERROR_CODE = "DatabaseError";
```

Comment:
```suggestion
    public static final String DATABASE_ERROR = "DatabaseError";
```
Shall we remove the CODE part? WDYT?